docs/book/src/user/troubleshooting.md

```diff
@@ -1,6 +1,16 @@
 # Troubleshooting
 
-## Node bootstrap failures when using CABPK with cloud-init
+## Interpretting token refresh messages
+
+Sometimes, you'll see that the bootstrapping secret is noted as not having been consumed yet, *even though* you correctly ran the bootstrapping phase on your node.  This happens because Cluster API didn't yet hear back from a VM that it expected to bootstrap and join the cluster.  In this case, you might see a recurrent:
+
+```
+Refreshing token until the infrastructure has a chance to consume it
+```
+
+message in the logs for your `capi-kubeadm-bootstrap-system` controller.  
```

Comment:
There's some context missing here as to the meaning of this message or the follow on. All of the headings are on the same level so it looks like there's no logical flow for solving this problem once ID'd - maybe add links or change the section headings.